core/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -42,6 +42,7 @@ public class TriggerHandler extends LifecycleAdapter implements TransactionEvent
     private enum Phase {before, after, rollback, afterAsync}
 
     public static final String TRIGGER_REFRESH = "apoc.trigger.refresh";
+    public static final String TRIGGER_PREVENT_PERSIST = "apoc.trigger.prevent.persist";
```

Comment:
Dovviamo sempre cercare di mettere property assertive
```suggestion
    public static final String TRIGGER_PERSIST = "apoc.trigger.persist";
```